packages/teams-js/jest-setup.cjs

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable strict-null-checks/all */
```

Comment:
The comment you put in the description about why this was needed was helpful, could you add a comment to the file so that if people look at this they know why it was added?